src/controllers/article.controller.js

```diff
@@ -0,0 +1,88 @@
+import BaseRepository from '../repository/base.repository';
+import responseGenerator from '../helpers/responseGenerator';
+import db from '../database/models';
+
+/**
+ * @class UserController
+ */
+class ArticleController {
+  /**
+   * Get all bookmarked articles
+   * @async
+   * @param {object} req - Request object
+   * @param {object} res - Response object
+   * @return {json} Returns json object
+   * @static
+   */
+  static async getBookMarks(req, res) {
+    // const { articleId } = req.body;
```

Comment:
please kindly remove this if you are not using it. @tolumide-ng

Reply:
Thank you, I would remove this